test/e2e/tests/metrics/developer-options-sentry.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { Suite } from 'mocha';
+import { withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { Driver } from '../../webdriver/driver';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import SettingsPage from '../../page-objects/pages/settings-page';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import DevelopOptions from '../../page-objects/pages/deveoper-options';
+import ErrorPage from '../../page-objects/pages/ErrorPage';
+
+const triggerCrash = async (driver: Driver): Promise<void> => {
+  const headerNavbar = new HeaderNavbar(driver);
+  await headerNavbar.openSettingsPage();
+  const settingsPage = new SettingsPage(driver);
+  await settingsPage.check_pageIsLoaded();
+  await settingsPage.goToDevelopOptionSettings();
+
+  const developOptionsPage = new DevelopOptions(driver);
+  await developOptionsPage.check_pageIsLoaded();
+  await developOptionsPage.clickGenerateCrashButton();
+};
+
+describe('Developer Options - Sentry', function (this: Suite) {
+  it('gives option to cause a page crash and provides sentry form to report', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        title: this.test?.fullTitle(),
+        ignoredConsoleErrors: ['ignore-all'],
```

Comment:
do you know if we can remove this, or alternatively, be more granular if there is just 1 or 2 errors appearing?
so we can catch new errors if those happen in the future. Ie

Reply:
Actually there were errors showing as i triggered the error manually on the page, removing will block the tests :( 
```
<testsuites name="Mocha Tests" time="23.228" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2024-10-17T22:24:45" tests="0" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Developer Options - Sentry" timestamp="2024-10-17T22:24:45" tests="2" file="/Users/danicashen/Documents/Consensys/MetaMask_Extension/metamask-extension/test/e2e/tests/metrics/developer-options-sentry.spec.ts" time="23.227" failures="1">
    <testcase name="Developer Options - Sentry gives option to cause a page crash and provides sentry form to report" time="11.250" classname="gives option to cause a page crash and provides sentry form to report">
    </testcase>
    <testcase name="Developer Options - Sentry gives option to cause a page crash and offer contact support option" time="11.958" classname="gives option to cause a page crash and offer contact support option">
      <failure message="The above error occurred in the &lt;SettingsPage&gt; component:
```

Reply:
Thanks for testing for me, this indeed looks much more specific, added your suggestion in [15be59](https://github.com/MetaMask/metamask-extension/pull/27857/commits/15be59ce4601178d8ce07bb4bf696799f94f455d)